src/utils/network_info.ts

```diff
@@ -408,6 +408,17 @@ const networks = {
     label: "Kava - KAVA",
     name: "Kava",
   },
+  "kava-testnet": {
+    address: "kavavaloper1vlpsrmdyuywvaqrv7rx6xga224sqfwz3yjnlkh",
+    delegate: "https://wallet.keplr.app/chains/kava", // FIXME test chain not found in keplr https://testnet.keplr.app/
+    denom: "KAVA",
+    graphql: "kava",
+    guide: "how-to-stake-akt-on-kava",
+    image: "/images/network/kava.png",
```

Comment:
This is fine, many images are png, although I've been moving some to svg, some logos will look a lot better, something to consider depending how they look

Reply:
Good to know.

I haven't added the asset though. I don't know from where to get the svg.

Reply:
Ok, will send a DM with the figma project with logos

Reply:
3e9852ab480bd5be4b5d9cab9053cb1695655518